### ISSUE TITLE
[Feature Request]: multi policies create

### ISSUE THREAD

**Issue by ibrahem-kamal** [laravel-idea/plugin#596]
### Feature Description

most of the project i work with i create some dynamic structure to read the policies and handle the roles/permissions and i found a way to modify the policy created to extend my structure within your documentation and that's great , but i still need to create the policy one by one  for each model so what i am asking for is a way to like select the full directory or multiple models at once and call the generate policy for them all

**Comment by adelf** [comment]
No. Anyway, you need to fill each policy with the actual code, so it doesn't make sense. Sorry.
